src/lazy/text/value.rs

```diff
@@ -145,51 +149,27 @@ impl<'data, D: TextEncoding<'data>> LazyRawValue<'data, D> for MatchedRawTextVal
     }
 }
 
-impl<'data> LazyRawValuePrivate<'data> for LazyRawTextValue_1_0<'data> {
```

Comment:
🗺️ Now that `*_1_0` and `*_1_1` share an implementation, I was able to delete this redundant `impl`.